sdk/framework/openapi.go

```diff
@@ -204,9 +213,9 @@ var (
 )
 
 // documentPaths parses all paths in a framework.Backend into OpenAPI paths.
-func documentPaths(backend *Backend, doc *OASDocument) error {
+func documentPaths(backend *Backend, requestResponsePrefix string, doc *OASDocument) error {
 	for _, p := range backend.Paths {
-		if err := documentPath(p, backend.SpecialPaths(), backend.BackendType, doc); err != nil {
+		if err := documentPath(p, backend.SpecialPaths(), requestResponsePrefix, backend.BackendType, doc); err != nil {
```

Comment:
Since we're changing this... As this has now accumulated 5 parameters, I wonder if it is worth considering a small refactor? Maybe `documentPath` could be a method on `*OASDocument`, and we could do like:

```go 
cfg := pathConfg{...}
if err := doc.appendPaths(cfg); err != nil {    // naming TBD
... 
}
```

Not a blocker, just an idea.

Reply:
I'd be fine either way. Probably cleanest as a separate PR.